{"name": "test_logout_forward[1846030109260034050-False--]", "status": "failed", "statusDetails": {"message": "AssertionError: 接口返回码是 【 200 】, 返回信息：操作成功 \nassert True == False\n +  where True = <api.result_api.ResultBase object at 0x0000029168A1A2B0>.success", "trace": "self = <testcases.api_test.index_uesr_test.test_03_logout.TestLogout object at 0x0000029168925610>\nlogin_test_user_fixture = 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJyYmFjLXVzZXIiLCJpc3MiOiJsb25nY2hhbyIsImlhdCI6MTczMDc5MzE4NCwiZXhwIjoxNzMwNzk2Nzg0LCJpZCI6MTg0OTM1MTM1NTc2NDA3NjU0Nn0.r1Beb5MKY-pGG2CPsnlaoNJf5WwAUD12XVUmKGJPNf0'\nuserId = 1846030109260034050, expect_result = False, expect_code = ''\nexpect_msg = ''\n\n    @allure.story(\"用例--用户登出\")\n    @allure.description(\"该用例是针对用户登出接口的测试\")\n    @pytest.mark.single\n    @pytest.mark.parametrize(\"userId, expect_result, expect_code, expect_msg\",\n                             api_data[\"test_logout\"])\n    def test_logout_forward(self, login_test_user_fixture,userId, expect_result, expect_code, expect_msg):\n        logger.info(\"*************** 开始执行用例 ***************\")\n        result = logout(userId)\n>       assert_results(result,expect_result, expect_code, expect_msg)\n\ntest_03_logout.py:21: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nresult = <api.result_api.ResultBase object at 0x0000029168A1A2B0>\nexpect_result = False, expect_code = '', expect_msg = ''\n\n    def assert_results(result, expect_result, expect_code, expect_msg):\n        \"\"\"\n        断言结果并记录日志。\n        参数:\n            result: 实际结果对象，应该包含success, error, data等属性。\n            expect_result: 期望的成功状态。\n            expect_code: 期望的返回code。\n            expect_msg: 期望的返回message。\n        \"\"\"\n    \n        # 断言实际结果与期望结果相同\n>       assert result.success == expect_result, result.error\nE       AssertionError: 接口返回码是 【 200 】, 返回信息：操作成功 \nE       assert True == False\nE        +  where True = <api.result_api.ResultBase object at 0x0000029168A1A2B0>.success\n\n..\\..\\conftest.py:25: AssertionError"}, "description": "该用例是针对用户登出接口的测试", "attachments": [{"name": "log", "source": "24b2bf8a-a7fb-45e9-9da9-87f3381626e5-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "userId", "value": "1846030109260034050"}, {"name": "expect_result", "value": "False"}, {"name": "expect_code", "value": "''"}, {"name": "expect_msg", "value": "''"}], "start": 1730793184363, "stop": 1730793184458, "uuid": "a45a8432-8619-4467-aef9-5ff99d603ba7", "historyId": "25f69c971e7c49df45a45ddbbef8b14c", "testCaseId": "43065d76d0638e161cc2ec96221a85a6", "fullName": "testcases.api_test.index_uesr_test.test_03_logout.TestLogout#test_logout_forward", "labels": [{"name": "feature", "value": "用户公共基础模块"}, {"name": "epic", "value": "针对单个接口的测试"}, {"name": "story", "value": "用例--用户登出"}, {"name": "tag", "value": "single"}, {"name": "parentSuite", "value": "testcases.api_test.index_uesr_test"}, {"name": "suite", "value": "test_03_logout"}, {"name": "subSuite", "value": "TestLogout"}, {"name": "host", "value": "CQCA202413058N"}, {"name": "thread", "value": "26000-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.api_test.index_uesr_test.test_03_logout"}]}